src/Extension/FluentExtension.php

```diff
@@ -636,7 +637,18 @@ public function onAfterWrite(): void
      */
     public function onAfterDuplicate($original, $doWrite, $relations): void
     {
+        $owner = $this->getOwner();
         $localisedTables = $this->owner->getLocalisedTables();
+        // Get the names of all has_one columns that will have new IDs
+        $copyRelations = (array) $owner->config()->get('localised_copy');
```

Comment:
Worth adding a check for `localisedCopyActive`?

Reply:
I'm not sure what that is or how it works. Can you please check if this is also a problem when that is used, and if so, add the necessary context and replication steps to the issue?

Reply:
Had a more detailed look on this one and I think it's fine to not have it as the intended case doesn't apply here.
 Context: this property allows you do avoid content duplication when models are being localised for some edge cases like in unit tests. This is not relevant for duplication scenario when a new model is being created.